src/conduits/java/com/enderio/conduits/common/init/ConduitItems.java

```diff
@@ -0,0 +1,57 @@
+package com.enderio.conduits.common.init;
+
+import com.enderio.EnderIO;
+import com.enderio.api.conduit.ConduitItemFactory;
+import com.enderio.api.conduit.IConduitType;
+import com.enderio.base.common.init.EIOCreativeTabs;
+import com.enderio.base.common.init.EIOItems;
+import com.enderio.conduits.common.items.ConduitBlockItem;
+import com.tterrag.registrate.Registrate;
+import com.tterrag.registrate.util.entry.ItemEntry;
+import com.tterrag.registrate.util.nullness.NonNullSupplier;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.item.Item;
+import net.minecraftforge.event.CommandEvent;
+import net.minecraftforge.event.entity.EntityJoinLevelEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.registries.ForgeRegistries;
+import java.util.function.Supplier;
+
+@Mod.EventBusSubscriber
+public class ConduitItems {
+    private static final Registrate REGISTRATE = EnderIO.registrate();
+
+    public static final ItemEntry<Item> POWER = createConduitItem(() -> EnderConduitTypes.POWER.get(), "power1");
+    public static final ItemEntry<Item> POWER2 = createConduitItem(() -> EnderConduitTypes.POWER2.get(), "power2");
+    public static final ItemEntry<Item> POWER3 = createConduitItem(() -> EnderConduitTypes.POWER3.get(), "power3");
```

Comment:
Going to keep the 3 tiers here for now, but I believe consensus was we're going to rework power and remove transfer caps, is that right @agnor99?